app/api/process.py

```diff
@@ -0,0 +1,95 @@
+"""
+The main "processing" module.
+"""
+
+import flask
+import numpy as np
+import pandas as pd
+from time import time
+
+from app.api import utils, ltc, aggregate_outbreaks, close_outbreaks, data_quality_checks, \
+    check_cumulative, unreset_cumulative
+
+
+_FUNCTION_LISTS = {
+    'CO': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'DE': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'FL': [
+        utils.standardize_data,
+        aggregate_outbreaks.preclean_FL,
+        aggregate_outbreaks.collapse_outbreak_rows,
+        aggregate_outbreaks.postclean_FL,
+        ],
+    'IL': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'ME': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'MN': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'ND': [
+        utils.standardize_data,
+        lambda df: aggregate_outbreaks.collapse_outbreak_rows(df, add_outbreak_and_cume=False),
+        ],  ## CHECK THIS
+    'NJ': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'OR': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+    'WY': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows],
+}
+
+
+def get_all_state_urls():
+    url_link = 'https://docs.google.com/spreadsheets/d/1kBL149bp8PWd_NMFm8Gxj-jXToSNEU9YNgQs0o9tREs/gviz/tq?tqx=out:csv&sheet=State_links'
```

Comment:
Should we put this in `config.py` just so it's not a bare URL deep in the app?

Reply:
is it okay to leave this as a TODO? there will be much moving around and deleting of code if we go with this approach in general

Reply:
Absolutely!